Testscripts/Linux/CORE-InitrdModulesCheck.sh

```diff
@@ -27,8 +27,8 @@ SearchModules()
     elif [[ -d "/usr/lib64/modules" ]]; then
         abs_path="/usr/lib64/modules/"
     fi
-    if [[ $DISTRO == "suse_12" ]]; then
-        abs_path="/lib/modules/"
+    if [[ $DISTRO == "suse_12" ]] || ([[ $DISTRO_NAME == "ubuntu" ]] && [[ $DISTRO_VERSION == "19.10" ]]); then
```

Comment:
SLES 12 may break the code. Please check.

Reply:
/lib/modules/4.12.14-6.23-azure/ in SLES 12 exists.
```
[LISAv2 Test Results Summary]
Test Run On           : 11/04/2019 02:05:52
ARM Image Under Test  : SUSE : SLES : 12-SP4 : latest
Initial Kernel Version: 4.12.14-6.23-azure
Final Kernel Version  : 4.12.14-6.23-azure
Total Test Cases      : 1 (0 Passed, 0 Failed, 0 Aborted, 1 Skipped)
Total Time (dd:hh:mm) : 0:0:3

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 CORE                 INITRD-MODULES-CHECK                                                           SKIPPED                 0.38 
```

Reply:
Cool